### ISSUE TITLE
Variable font assets

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/cloudfour.com-patterns#469]
Although our efforts to add Source Sans Variable to the current iteration have been somewhat frustrating (as documented in #452, which you should totally read before starting this). A major release gives us an opportunity to revise our strategy for this.

To keep us from stalling completely, I will be adding a separate issue for font loading strategies. This issue should prioritize getting it working in a simple way that sets us up for future success.

## Tasks

- [ ] Generate/procure/include necessary font files
  - [ ] Source Sans Variable (body copy)
    - [ ] WOFF2 / Regular
    - [ ] WOFF2 / Italic
  - [ ] Source Sans Pro (fallback where variable fonts aren't supported)
    - [ ] WOFF2
      - [ ] Regular
      - [ ] Semibold
      - [ ] Bold
      - [ ] Italic
    - [ ] WOFF
      - [ ] Regular
      - [ ] Semibold
      - [ ] Bold
      - [ ] Italic
  - [ ] Source Sans Code (code samples)
    - [ ] WOFF2 / Regular
    - [ ] WOFF / Regular
- [ ] Add CSS/Sass necessary for usage
